DashAI/front/src/components/experiments/RunnerDialog.jsx

```diff
@@ -40,9 +44,29 @@ function RunnerDialog({ experiment, expRunning, setExpRunning }) {
         return { ...run, status: getRunStatus(run.status) };
       });
       setRows(runsWithStringStatus);
+
       if (rowSelectionModel.length === 0) {
         setRowSelectionModel(runs.map((run, idx) => run.id));
       }
+
+      const running = runs.find((run) => run.id === runningNow);
+
+      if (running && running.status === 3) {
+        const allRunsFinished = runs
+          .filter((run) => rowSelectionModel.includes(run.id))
+          .every((run) => run.status === 3);
+        if (allRunsFinished) {
+          setExpRunning({ ...expRunning, [experiment.id]: false });
+          enqueueSnackbar(
+            `${experiment.name} has succesfully finished running`,
```

Comment:
Maybe:

```suggestion
            `${experiment.name} has completed all its runs.`,
```

What happen with this message if some run fails?
It also prints the same message?

Reply:
I was only handling request errors, if there was an error in the model execution it would loop because that message was displayed only when all statuses were "finished".

I included the "Error" status in the checks and changed the message.